x/leverage/keeper/collateral.go

```diff
@@ -112,17 +139,16 @@ func (k Keeper) CollateralLiquidity(ctx sdk.Context, denom string) sdk.Dec {
 	return sdk.MinDec(collateralLiquidity, sdk.OneDec())
 }
 
-// CollateralShare calculates the portion of overall collateral
-// (measured in USD value) that a given uToken denom represents.
-func (k *Keeper) CollateralShare(ctx sdk.Context, denom string) (sdk.Dec, error) {
+// VisibleCollateralShare calculates the portion of overall collateral (measured in USD value) that a
+// given uToken denom represents. If an asset other than the denom requested is missing an oracle
+// price, it ignores that asset's contribution to the system's overall collateral, thus potentially
+// overestimating the requested denom's collateral share while improving availability.
+func (k *Keeper) VisibleCollateralShare(ctx sdk.Context, denom string) (sdk.Dec, error) {
```

Comment:
Do we need to rename this function? I would prefer to not rename it if possible.